.travis.yml

```diff
@@ -12,7 +12,7 @@ addons:
     packages:
       - python2.4
 install:
-  - pip install tox PyYAML Jinja2 sphinx
+  - pip install tox PyYAML Jinja2 sphinx netaddr
```

Comment:
Then how to write tests for i.e. core filter plugins?
Isn't netaddr needed to test the core filter plugin?
Should the tests be dropped?